### DISCUSSION TITLE
Discern signers from previous multi-sig transaction?

### DISCUSSION THREAD

**Discussion by bitcoincoredotbin** [cryptoadvance/specter-desktop#981] [General]
Is there a way to look at a previous transaction and determine which signers were used to complete the transaction? Rather, which one was excluded in a 2of3 scheme?

Thanks!

**Comment by ben-kaufman** [comment]
Hi, there's no such option currently, but I think it would be a very nice feature to have, so will try to get there and implement it (hopefully soon)
